.github/workflows/test_optimizer.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
   build_hooks:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
     with:
-      dart_sdk: 2.19.0
+      dart_sdk: 2.18.0
```

Comment:
I think you will need to revert this in order to use Very Good Analysis 4.0.0

Reply:
```suggestion
      dart_sdk: 2.19.0
```

Reply:
Who is this Felix guy